tests/visualization_tests/test_edf.py

```diff
@@ -11,6 +14,15 @@ def test_target_is_none_and_study_is_multi_obj() -> None:
         plot_edf(study)
 
 
+def _validate_edf_values(edf_values: Sequence[float]) -> None:
+    np_values = np.array(y_values)
+    # This line confirms that the values are monotonically non-decreasing.
+    assert np.all(np_values[1:] - np_values[:-1] >= 0)
+
+    # Confirms that the values are in [0,1].
+    assert np.all((0 <= np_values) * (np_values <= 1))
```

Comment:
`&` is recommended for logical and operation.
🔗 https://numpy.org/doc/stable/reference/generated/numpy.logical_and.html

```suggestion
    assert np.all((0 <= np_values) & (np_values <= 1))
```